tests/src/common/TestUtils.java

```diff
@@ -326,4 +332,34 @@ public static RunResult runCmd(int expectedExitCode, int timeoutMilli, File dir,
         rr.validateExitCode(expectedExitCode);
         return rr;
     }
+
```

Comment:
should there be  comment or warning to these methods since they are not used - they encourage doing things by copy/restore of the prop file which i think we want to discourage.